test/verify/check-system-info

```diff
@@ -686,19 +686,32 @@ fi
         m.write("/usr/share/cockpit/subscription-manager/override.json", '{ "features": { "insights": true } }')
 
         # Run a mock version of the Insights API locally and configure
-        # insights-client to access it.
-
-        m.upload(["verify/files/mock-insights"], "/var/tmp")
+        # insights-client to access it. That requires a good enough
+        # TLS mock insights server certificate
+        m.upload(["verify/files/mock-insights", "../src/tls/ca/ca.conf", "../src/tls/ca/ca.key", "../src/tls/ca/ca.pem"], "/var/tmp")
+        m.execute("""set -e
+cd /var/tmp
+openssl genrsa -out insights.key 2048
+openssl req -new -key insights.key -out insights.csr -config ca.conf -subj "/CN=localhost/DC=COCKPIT/DC=LAN/"
+mkdir certs
+touch index.txt
+echo 01 > serial
+openssl ca -batch -config ca.conf -in insights.csr -days 365 -notext -extensions usr_cert -out insights.pem -subj "/CN=localhost/DC=COCKPIT/DC=LAN/"
+
+cp ca.pem /etc/pki/ca-trust/source/anchors/
+update-ca-trust
+""")
         m.spawn("/var/tmp/mock-insights", "mock-insights")
         m.write("/etc/insights-client/insights-client.conf",
                 """
 [insights-client]
 gpg=False
 auto_config=False
-base_url=127.0.0.1:8888/r/insights
+base_url=localhost:8888/r/insights
+# HACK: despite the good insights.cert above, some later part of insights *still* does not respect that
+cert_verify=False
```

Comment:
Do we need all the cert fun if we just then ignore it?

Reply:
The server still needs to load a TLS certificate which makes the initial connection not hiccup. I started with that, and waded through all the follow-up errors before I gave up there. I can try to simplify it a bit.

Reply:
See e.g. [this first attempt](https://github.com/cockpit-project/bots/pull/1921#issuecomment-820242174) with a simple certificate, which was already doomed. It really doesn't want to be that easy :cry:

Reply:
Indeed it works with the plain "alice" certificate, yay! Much simpler. Dang, I did this the wrong way around..